### PR TITLE
Some optimizations for dashboard handlers

### PR DESCRIPTION
**Pull request by powerfooI**
## Summary

1. Modified timestamp fields of `float64` type to `int64`
2. Renamed package `internal/oceanbase` to `internal/clients` for concision
3. Added events fields for backup policy that contains events of itself and its sub jobs
4. Added logging outputs for all gin handlers
5. Removed empty README in `ui/` folder
6. Fixed metrics plots with different amount of points by interpolation

### CODE REVIEW

File: internal/dashboard/business/metric/metric.go [hunk 1]
```diff
@@ -104,13 +105,35 @@ func extractMetricData(name string, resp *external.PrometheusQueryRangeResponse)
...
+		}
+		lenValues := len(values)
+		if lenValues == 0 {
+			continue
+		}
+		for i := range values {
+			// interpolate zero slot with average of previous and next value
+			if values[i].Value == 0 {
+				switch i {
+				case 0:
```
Comment (chris-sun-star) [inline internal/dashboard/business/metric/metric.go:126]: is it possible to solve the problem in front end, it's really normal to lose some datapoint

File: internal/dashboard/business/metric/metric.go [hunk 2]
```diff
@@ -104,13 +105,35 @@ func extractMetricData(name string, resp *external.PrometheusQueryRangeResponse)
...
+					if lenValues > 1 {
+						values[i].Value = values[i+1].Value
+					}
+				case lenValues - 1:
+					if lenValues > 1 {
+						values[i].Value = values[i-1].Value
+					}
+				default:
```
Comment (chris-sun-star) [inline internal/dashboard/business/metric/metric.go:134]: what if more than one data point is lost, values[i+1] is 0, then values[i] equals values[i-1].Value / 2

File: internal/dashboard/router/v1/obcluster_router.go [hunk 3]
```diff
@@ -29,4 +29,5 @@ func InitOBClusterRoutes(g *gin.RouterGroup) {
 	g.POST("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName/scale", h.Wrap(h.ScaleOBServer))
 	g.DELETE("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName", h.Wrap(h.DeleteOBZone))
 	g.GET("/obclusters/:namespace/:name/essential-parameters", h.Wrap(h.ListOBClusterResources))
```
Comment (chris-sun-star) [inline internal/dashboard/router/v1/obcluster_router.go:31]: don't forge to delete this line after front end modifies the url

Comment (PowerfooI) [reply internal/dashboard/router/v1/obcluster_router.go:31]: Oh yes, but the change need to be noticed by frontend app.

File: internal/dashboard/business/metric/metric.go (hunk 2 above)
Comment (PowerfooI) [reply internal/dashboard/business/metric/metric.go:134]: I think it doesn't matter. It's a tradeoff which is better than hollow NaNs. Interpolation has a better rendering effects.

File: internal/dashboard/business/metric/metric.go (hunk 1 above)
Comment (PowerfooI) [reply internal/dashboard/business/metric/metric.go:126]: No, I asked Yangon. The component he used in frontend can not handle the NaN. Interpolation is an inevitable choice.

File: internal/dashboard/business/metric/metric.go (hunk 2 above)
Comment (PowerfooI) [reply internal/dashboard/business/metric/metric.go]: I refined the interpolation logic and now it concerns continuous NaNs.

**Review by chris-sun-star — APPROVED** [review]

**Pull request merged**